applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_element.hpp

```diff
@@ -230,7 +230,9 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) UPwSmallStrainElement : public UPwBa
 
     void CalculatePermeabilityUpdateFactor(ElementVariables& rVariables);
 
-    virtual void CalculateBMatrix(Matrix& rB, const Matrix& GradNpT, const Vector& Np);
+    void                CalculateBMatrix(Matrix& rB, const Matrix& GradNpT, const Vector& Np) const;
+    std::vector<Matrix> CalculateBMatrices(const Matrix& NContainer,
+                                           const GeometryType::ShapeFunctionsGradientsType& DN_DXContainer) const;
```

Comment:
Probably only confusion for me:
the stress state policies have a CalculateBMatrix which return a matrix and take the 3 arguments rGradNpT, rNp and rGeometry
the elements have a CalculateBMatrix with return type void and 3 arguments rB, GradNpT and Np
and a CalculateBMatrices which returns a std::vector\<Matrix\> and 2 arguments NContainer and DN_DXContainer

The naming and order of the arguments is confusing for me. Please have the same order for GradNpT and Np 
GradNpT and the elements of DN_DXContainer are the same thing.
The p I don't understand, we look at the N for displacement here, so Nu or just N

Reply:
Done